README.md

```diff
@@ -37,6 +37,26 @@ Additionally when a section `sharding_key` is not empty in YAML schema call of
 function `ddl.set_schema(schema)` creates a space `_ddl_sharding_key` with two
 fields: `space_name` with type `string` and `sharding_key` with type `array`.
 
+Similarly for `sharding_func`: when a section `sharding_func` is not empty in YAML schema call of
+function `ddl.set_schema(schema)` creates a space `_ddl_sharding_func` with three
+fields: `space_name` with type `string`, `sharding_func` with type `string` and
+`lua_code` with type `boolean`.
+
+These sharding functions are supported now:
+
+- `strcrc32`;
+- `mpcrc32`.
+
+You can use short names like `strcrc32`/`mpcrc32` 
+or full names `vshard.router.bucket_id_strcrc32()`/`vshard.router.bucket_id_mpcrc32()`.
+
+Also you can pass your own sharding function using lua code:
+```lua
+sharding_func = {
+  body = 'function(key) return <...> end'
+}
```

Comment:
We need to explicitly specify what sharding function can accept (number of parameters and their types) and what function must return (number of parameters and their types).

Reply:
Updated

Reply:
It looks resolved. @ligurio?